.gitignore

```diff
@@ -2,3 +2,5 @@ vendor/
 composer.lock
 phpunit.xml
 .php_cs.cache
+.idea
```

Comment:
I think IDE folders should be part of the global`.gitignore` file instead of modifying this file at project level

Reply:
Sure? It seems to me this contains quite personal configuration settings, like window size, position etc. Not sure everyone would want it that way.